partner_contact_in_several_companies/models/res_partner.py

```diff
@@ -0,0 +1,161 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models, _
+from odoo.osv import expression
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    contact_type = fields.Selection(
+        [('standalone', _('Standalone Contact')),
```

Comment:
No need for _("something") in selection. Strings are automatically available for translation.